### PR TITLE
Introduction Submit

### PR DESCRIPTION
**Pull request by NathanBland**
Hey now that there is an option that fits my standing I can submit this ;)

### CODE REVIEW

File: introduction.js [hunk 1]
```diff
@@ -1 +1,10 @@
 // Your code goes here.
+var name = 'Nathan bland'
+var age = 23
+var major = 'C.S.'
```
Comment (barberboy) [inline introduction.js]: Do you mind spelling this out? `C.S.` is a bit ambiguous.

File: introduction.js [hunk 2]
```diff
@@ -1 +1,10 @@
 // Your code goes here.
+var name = 'Nathan bland'
```
Comment (barberboy) [inline introduction.js]: Please capitalize your last name.

**Comment by barberboy** [comment]
This looks good aside from the two issues mentioned in the diff. Fix those up and commit/push, and it looks good to me!

**Comment by NathanBland** [comment]
I suppose `C.S.` could mean something else? Though I'm not sure what.
Fixed the name typo as well.. So picky...

**Comment by barberboy** [comment]
Mostly just wanting to utilize Line comments, but yes I'm quite picky ;)

Looks great! :100:

**Pull request closed without merging**